fiftyone/core/tags.py

```
@@ -31,3 +49,1496 @@
+        "end": end,
+        "tag": tag,
+    }
+
+
+def _get_or_create_collection():
+    collection = foo.get_db_conn()[TAGS_COLLECTION_NAME]
+    _ensure_indexes(collection)
+    return collection
+
+
+def _get_existing_collection():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_get_existing_collection`

(ANN202)